source/features/fit-textareas.css

```diff
@@ -1,4 +1,5 @@
 .rgh-fit-textareas {
+	height: auto !important;
```

Comment:
I believe this will break the manual resizing (you can drag the bottom right corner to resize it)

Reply:
indeed it does 🤔 😟

the reason I added it was that:

1. the `height: 100px !important` below
2. `height: <pixel value>` was appearing as an inline style on some of these `textarea`s

```css
.issue-form-textarea {
    height: 100px !important;
    min-height: 100px !important;
}
```

but maybe I should just replace that `.issue-form-textarea` class instead (and probably others, eg. PRs, Discussions, etc, hmm...

if you have any suggestions, also happy to look

Reply:
in addition to the styles above, there's also another block for the unfocused fields:

```css
.form-group textarea.form-control {
    width: 100%;
    height: 200px;
    min-height: 200px;
}
```

where the `height: 200px` is also causing constraints there

Reply:
alright, I tried out the first approach:

- remove the first class 84903c6cfa4e35b5ba9253159e9b4205795d9df1
- make selector stronger 8e890d62f73550e1621396712952b77f5e8c8488

Reply:
if the `height: auto !important` with the `.rgh-fit-textarea.issue-form-textarea` selector is preferable after all, I can switch to that instead